rest-api/src/main/java/gov/cms/qpp/conversion/api/services/VersionService.java

```diff
@@ -0,0 +1,7 @@
+package gov.cms.qpp.conversion.api.services;
+
+public interface VersionService {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Document this public interface by adding an explicit description. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUndocumentedApi)